hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -106,6 +106,11 @@
     ],
   },
 
+  // List of unmanaged external resets
+  unmanaged_resets: [
+    // { name: "my_ext" }
+  ]
+
```

Comment:
I don't really think we need to document the variable here (where it's getting defined for a particular toplevel). If we do want to document it, I'd probably suggest filling out the example more explicitly, but not in the list. Maybe something like this? (in the template)
```
  // List of unmanaged external resets
  //
  // These should be listed as dictionaries that just give their names, in this form:
  //
  //    { name: "my_external_reset" }
  unmanaged_resets: []
```

Reply:
I removed that at all, though I'll add that for Darjeeling?